benchmark/fluid/Dockerfile

```diff
@@ -1,11 +1,17 @@
 FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+
+ARG UBUNTU_MIRROR
```

Comment:
Not all versions of docker support `ARG`. And change one line of Dockerfile seems easier?